ha-addon/mqtt_discovery/izar.json

```diff
@@ -0,0 +1,148 @@
+{
+  "total_m3": {
+    "component": "sensor",
+    "discovery_payload": {
+      "device": {
+        "identifiers": ["wmbusmeters_{id}"],
+        "manufacturer": "Diehl",
+        "model": "Water meter IZAR RC 868 I R4 PL",
+        "name": "{name}",
+        "sw_version": "{id}"
+      },
+      "name": "{name} total",
+      "unique_id": "wmbusmeters_{id}_{attribute}",
+      "state_topic": "wmbusmeters/{name}",
+      "value_template": "{{ value_json.{attribute} }}",
+      "json_attributes_topic": "wmbusmeters/{name}",
+      "icon": "mdi:gauge",
+      "unit_of_measurement": "m³",
+      "state_class": "total_increasing",
+      "device_class": "water",
+      "enabled_by_default": true
+    }
+  },
+
+  "last_month_total_m3": {
+    "component": "sensor",
+    "discovery_payload": {
+      "device": {
+        "identifiers": ["wmbusmeters_{id}"],
+        "manufacturer": "Diehl",
+        "model": "Water meter IZAR RC 868 I R4 PL",
+        "name": "{name}",
+        "sw_version": "{id}"
```

Comment:
it is not necessary to use the field MQTT **sw_version** since we do not have this information on the statements